tools/codegen/dest/native_functions.py

```diff
@@ -22,12 +25,21 @@ def gen_unstructured(f: NativeFunction) -> List[str]:
         if "legacy::" in n:
             continue
         seen.add(n)
-        returns_type = native.returns_type(f.func.returns).cpp_type()
-        args = native.arguments(f.func)
-        rs.append(f"TORCH_API {returns_type} {n}({', '.join(a.decl() for a in args)});")
+        rs.append(f"TORCH_API {native_sig.decl(name=n)};")
```

Comment:
This refactor seems like it could have been done in a prep commit. Maybe not that long so not worth it.

Reply:
yep it's totally unrelated. I noticed it and made the change after I'd already made a bunch of changes to this file, so I thought it would be more effort than it was worth to split up. But that might be saying something about my workflow more than anything else.